apiserver/main.go

```diff
@@ -15,6 +16,25 @@ var (
 	logger                         *config.Tracelog
 )
 
+func createActiveProfileDirectories(activeProfile config.DeviceInfo) {
+	var pathErr error
+	// Create /var/www/ekstep/config_json dir
```

Comment:
Better to add what ConfigJsonDir contain, rater than /var/www/ekstep/config_json dir..